tests/conftest.py

```diff
@@ -212,6 +220,25 @@ def dummy_litter_data(layer_roles_fixture):
         )
     )
 
+    # The layer dependant data has to be handled separately
+    data["soil_moisture"] = xr.concat(
+        [
+            DataArray(np.full((13, 3), np.nan), dims=["layers", "cell_id"]),
+            # At present the soil model only uses the top soil layer, so this is the
+            # only one with real test values in
+            DataArray([[0.5, 0.7, 0.6]], dims=["layers", "cell_id"]),
+            DataArray(np.full((1, 3), 0.1), dims=["layers", "cell_id"]),
```

Comment:
Why/how did you set the value for the lowest soil layer?

Reply:
Uhh, I copied and pasted it from `dummy_carbon_data` 😅. I don't actually use the lowest soil layer for anything, would it be better for this layer to just be `nan`?

Reply:
maybe yes, at least for now